scripts/cli2.0.js

```diff
@@ -0,0 +1,80 @@
+const inquirer = require('inquirer');
+const argv = require('minimist')(process.argv.slice(2), {
+    alias: { lib: ['library', 'l'], type: 't' },
+} );
+const fs = require('fs-extra');
+const path = require('path');
+
+const libraries = [
+    'eris',
+    'discordjs',
+    'discord.js',
+];
+const type = [
+    'esm',
```

Comment:
maybe we can rename `esm` to `esmjs` then?

Reply:
This will mean I need to rename every folder and I don't fancy doing that :P

Reply:
fair point

Reply:
we could just make the option esmjs without renaming the folders but we would lack consistency

Reply:
It would also mean I need to do some whacky stuff with the inquirer

Reply:
yeah that's fair, let's keep it like that for now